application/views/admin/participants/summary_view.php

```diff
@@ -17,48 +17,54 @@
         <div class="row">
             <div class="col-12 content-right table-responsive">
                 <table class='ls-statisticssummary table table-hover'>
+                    <caption class="sr-only">Central participants database summary</caption>
                     <tr>
-                        <th>
+                        <th scope="row">
                             <?php eT("Total participants in central table"); ?>
                         </th>
                         <td>
                             <?php echo $totalrecords; ?>
                         </td>
                     </tr>
                     <tr>
-                        <th>
+                        <th scope="row">
                             <?php eT("Participants owned by you"); ?>
                         </th>
                         <td>
                             <?php echo $owned . ' / ' . $totalrecords; ?>
                         </td>
                     </tr>
                     <tr>
-                        <th>
+                        <th scope="row">
                             <?php eT("Participants shared with you"); ?>
                         </th>
                         <td>
                             <?php echo $totalrecords - $owned . ' / ' . $totalrecords; ?>
                         </td>
                     </tr>
                     <tr>
-                        <th>
+                        <th scope="row">
                             <?php eT("Participants you have shared"); ?>
                         </th>
                         <td>
                             <?php echo $shared . ' / ' . $totalrecords; ?>
                         </td>
                     </tr>
                     <tr>
+<<<<<<< HEAD
```

Comment:
and here....

Reply:
Yes, I see that now. On my local system it shows I have merged without any conflicts, but it seems like it is not done correctly. I will delete this pull request and do it again. Sorry for the inconvenience